server/src/main/java/org/candlepin/model/Owner.java

```diff
@@ -442,4 +470,69 @@ public Boolean getAutobindDisabled() {
     public void setAutobindDisabled(Boolean autobindDisabled) {
         this.autobindDisabled = autobindDisabled;
     }
+
+    /**
+     * Utility method that defaults to 'entitlement'.
+     *  This covers legacy owners
+     *
+     * @return access mode.
+     */
+    @XmlTransient
+    public String contentAccessMode() {
+        return StringUtils.isEmpty(getContentAccessMode()) ?
+                ContentAccessCertServiceAdapter.DEFAULT_CONTENT_ACCESS_MODE : getContentAccessMode();
+    }
+
+    /**
+     * Returns the value of the contentAccessMode setting.
+     *
+     * @return String the value
+     */
+    public String getContentAccessMode() {
+        return contentAccessMode;
+    }
+
+    public void setContentAccessMode(String contentAccessMode) {
+        this.contentAccessMode = contentAccessMode;
+    }
+
+    /**
+     * Returns the value of the contentAccessModeList setting.
+     *
+     * @return String the value
+     */
+    public String getContentAccessModeList() {
+        return this.contentAccessModeList;
+    }
+
+    public void setContentAccessModeList(String contentAccessModeList) {
+        this.contentAccessModeList = contentAccessModeList;
+    }
+
+    @XmlTransient
+    public boolean hasContentAccessMode(String mode) {
+        if (StringUtils.isEmpty(mode)) {
+            return true;
```

Comment:
Is this a valid case? Does null/empty represent an actual state? Maybe this only seems weird to me because of the verbiage of the method name.